ts/src/zebpay.ts

```diff
@@ -0,0 +1,1933 @@
+        if (type !== 'spot') {
+            throw new NotSupported (this.id + ' fetchTickers() does not support ' + type + ' markets');
+        }
+        await this.loadMarkets ();
+        const request: Dict = {};
+        params['defaultType'] = 'spot';
+        const response = await this.publicSpotGetMarketAllTickers (this.extend (request, params));
+        //
+        //     [
+        //        {
+        //            "symbol": "BTC-INR",
+        //            "bestBid": "4900000",
+        //            "bestBidQty": "0.00014938",
+        //            "bestAsk": "",
+        //            "bestAskQty": "0",
+        //            "priceChange": "-98134.56",
+        //            "priceChangePercent": "-1.84",
+        //            "high": "5433400",
+        //            "low": "5333400",
+        //            "vol": "0.0002",
+        //            "volValue": "1066.68",
+        //            "last": "5333400"
+        //        }
+        //     ]
+        //
+        const tickerList = this.safeList (response, 'data', []);
+        return this.parseTickers (tickerList, undefined);
+    }
+
+    /**
+     * @method
+     * @name zebpay#fetchOHLCV
+     * @description fetches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+     * @see https://github.com/zebpay/zebpay-api-references/blob/main/spot/api-reference/public-endpoints.md#get-klinescandlesticks
+     * @param {string} symbol unified symbol of the market to fetch OHLCV data for
+     * @param {string} timeframe the length of time each candle represents
+     * @param {int} [since] timestamp in ms of the earliest candle to fetch
+     * @param {int} [limit] the maximum amount of candles to fetch
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @param {int} [params.endtime] the latest time in ms to fetch orders for
+     * @returns {int[][]} A list of candles ordered as timestamp, open, high, low, close, volume
+     */
+    async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = undefined, limit: Int = undefined, params = {}): Promise<OHLCV[]> {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const endtime = this.safeNumber (params, 'endTime');
+        params = this.omit (params, [ 'endtime' ]);
+        const request: Dict = {};
+        if (limit === undefined) {
+            limit = 200; // default is 200
+        }
+        let response = undefined;
+        params['defaultType'] = market['type'];
+        if (market['spot']) {
+            params['symbol'] = market['id'];
+            params['startTime'] = since;
+            params['limit'] = limit; // max 200, default 200
+            params['endTime'] = endtime;
+            params['interval'] = timeframe;
+            response = await this.publicSpotGetMarketKlines (this.extend (request, params));
```

Comment:
using `params[]` instead of `request[]` here. 
Also you're not checking if `since` is undefined before assigning it